app/demo-geant-integration/RunAction.cc

```diff
@@ -62,8 +62,9 @@ void RunAction::BeginOfRunAction(G4Run const* run)
                 = GlobalSetup::Instance()->GetGeometryFile();
         }
 
-        // Create the along-step action
-        GlobalSetup::Instance()->SetAlongStep(NoFieldAlongStepFactory{});
+        // Create the along-step action (no field)
+        GlobalSetup::Instance()->SetAlongStep(
+            celeritas::UniformAlongStepFactory{});
```

Comment:
Can we pass an actual value of a uniform magnetic field via a parameter?  Something like,
```
double z_value = GlobalSetup::Instance()->GetZField();
std::function<G4ThreeVector()> field = [] { return G4ThreeVector(0, 0, z_value); };
GlobalSetup::Instance()->SetAlongStep(celeritas::UniformAlongStepFactory(field));
```

Reply:
Oh actually the way we're doing it in `accel` we should be able to. Because of some of the weirdness of initialization ordering (e.g. the user macro file can change the field and immediately call `beamOn`) a lot of stuff has to be "lazily" constructed. I'll clean this up a bit to make the future extension to nonzero field more clear...